pkg/testing/ca/ca.go

```diff
@@ -284,7 +286,24 @@ func (ca *VirtualSigstore) GenerateTlogEntry(leafCert *x509.Certificate, envelop
 		return nil, err
 	}
 
-	return tlog.NewEntry(rekorBodyRaw, integratedTime, logIndex, rekorLogIDRaw, set, nil)
+	signer, err := signature.LoadECDSASignerVerifier(ca.rekorKey, crypto.SHA256)
+	if err != nil {
+		return nil, err
+	}
+	rootHash := sha256.Sum256(append([]byte("\000"), rekorBodyRaw...))
+	encodedRootHash := hex.EncodeToString(rootHash[:])
+	scBytes, err := util.CreateAndSignCheckpoint(context.TODO(), "rekor.localhost", int64(123), uint64(42), rootHash[:], signer)
+	if err != nil {
+		return nil, err
+	}
+	inclusionProof := &models.InclusionProof{
+		TreeSize:   swag.Int64(int64(1)),
+		RootHash:   &encodedRootHash,
+		LogIndex:   swag.Int64(0),
+		Hashes:     nil,
+		Checkpoint: swag.String(string(scBytes)),
+	}
+	return tlog.NewEntry(rekorBodyRaw, integratedTime, logIndex, rekorLogIDRaw, set, inclusionProof)
```

Comment:
This is now including an inclusion proof for all entries, which changes the verification path for all the tests that were using this helper. Could there be a flag to opt into generating the inclusion proof, and a separate test that opts into it, so that all the existing tests continue testing the path they were testing before?